app/search.py

```diff
@@ -256,7 +256,7 @@ def _swiss_search(self):  # pylint: disable=too-many-branches, too-many-statemen
 
             wildcard_results = results[0].get('matches', [])
             merged_results = []
-            if len(results) == 2:
+            if len(results) == 1:
```

Comment:
sure about that?
are the e2e tests happy with it?

Reply:
no, I will investigate - it is more the merge of both results that seems to have a bug...